libraries/rush-lib/src/cli/actions/BaseInstallAction.ts

```diff
@@ -153,6 +159,14 @@ export abstract class BaseInstallAction extends BaseRushAction {
     try {
       await installManager.doInstallAsync();
 
+      if (this._autoinstallersParameter.value) {
+        if (installManagerOptions.allowShrinkwrapUpdates) {
+          Autoinstaller.updateAutoinstallers(this.rushConfiguration);
```

Comment:
Sorta odd that this one is sync.

Reply:
Hmm... I'll see what happens if I make it async, might be worth it it to safeguard against future need for async methods.